src/site/_includes/layout.njk

```diff
@@ -148,7 +150,7 @@
     <main>
       {% include 'partials/banner.njk' %}
       <div id="content">
-        {{ content | safe }}
+        {% Partial page, collections, renderData %}{{ content | safe }}{% endPartial %}
```

Comment:
Might consider passing all of the `data` properties we care about to the shortcode. Then we don't need to loop over `collections`.

Reply:
Done.